### PR TITLE
Add code for GIANT gene networks analyses

### PR DESCRIPTION
**Pull request by miltondp**
This PR adds several modules and notebooks to replicate gene pairs highly ranked by correlation coefficients in tissue-specific gene networks from GIANT/HumanBase (https://hb.flatironinstitute.org/). The modules contain functions to access the REST API of HumanBase and predict a tissue-specific network given a pair of genes. The notebooks analyze and compare these results between CCC (Clustermatch Correlation Coefficient) and Pearson.

### CODE REVIEW

**Review by nrosed — APPROVED** [review]
Looks good, I added some comments regarding my lack of understanding about a few things and one plot question, but from what I understand it all looks good.

File: nbs/99_manuscript/giant/03_05-giant-predict_tissue.ipynb [hunk 1]
```diff
@@ -0,0 +1,1855 @@
+{
```
[the remark this answers is not in the document]
Comment (miltondp) [reply nbs/99_manuscript/giant/03_05-giant-predict_tissue.ipynb:1]: <p>Yes, good point. It's just internal (the numbers are correlation values), and only the gene pairs are actually used, but I will add it anyways.</p>

File: nbs/99_manuscript/giant/03_10-giant-gene_interactions_table.ipynb [hunk 2]
```diff
@@ -0,0 +1,1182 @@
+{
```
[the remark this answers is not in the document]
Comment (miltondp) [reply nbs/99_manuscript/giant/03_10-giant-gene_interactions_table.ipynb:1]: <p>Correlation is only performed in GTEx locally, and then we use GIANT to sort of "replicate" a high correlation in GTEx. Yes, we include everything from GIANT., and all the default values in the GIANT web page (15 max genes per network, suggested minimum cut weight, etc).</p><p> </p><p>I've added these comments in the notebook.</p>

[the remark this answers is not in the document]
Comment (miltondp) [reply nbs/99_manuscript/giant/03_10-giant-gene_interactions_table.ipynb:1]: <p>It's a python <a href="https://docs.python.org/3.9/library/glob.html" target="_blank">glob pattern</a>, similar to bash. <code>???</code> will match three characters. </p>

File: nbs/99_manuscript/giant/03_10-giant-summarize_tissue_predictions.ipynb [hunk 3]
```diff
@@ -0,0 +1,3297 @@
+{
```
[the remark this answers is not in the document]
Comment (miltondp) [reply nbs/99_manuscript/giant/03_10-giant-summarize_tissue_predictions.ipynb:1]: <p>Yes, done! It's explained in the manuscript, but it's good to have it here.</p>

[the remark this answers is not in the document]
Comment (miltondp) [reply nbs/99_manuscript/giant/03_10-giant-summarize_tissue_predictions.ipynb:1]: <p>No, the first plot versions are just drafts. These ones are the final ones, and are edited to add more info later (with Inkscape). But I added a comment describing the purpose here.</p>

**Pull request merged**